ios/FluentUI/Popup Menu/PopupMenuProtocols.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Defines the timing for the call of the onSelected closure/block
+@objc(MSFPopupMenuItemExecutionMode)
+public enum ExecutionMode: Int {
+    /// `onSelected` is called right after item is tapped, before popup menu dismissal
+    case onSelection
+    /// `onSelected` is called right after item is tapped, but prevent popup menu dismissal
+    case onSelectionWithoutDismissal
```

Comment:
I think it would be beneficial to add a demo app test case for this execution mode so in the future the chances of regression will be lower

Reply:
Done. Add a test case in Popup demo.
<img width="403" alt="image" src="https://user-images.githubusercontent.com/25728869/83882429-b0a6fc80-a774-11ea-9e9c-3b59f691ee3a.png">